docs/src/__configuration__/markdownMapping.tsx

```diff
@@ -0,0 +1,284 @@
+            TypographyProps={{ variant: 'body1', style: { fontWeight: 600 }, ...props }}
+        />
+    ),
+    h4: (props: TypographyProps): JSX.Element => (
+        <Headline
+            style={{ marginTop: 16 }}
+            hash={getHash(props.children?.toString() || 'h4')}
+            TypographyProps={{ variant: 'subtitle1', ...props }}
+        />
+    ),
+    h5: (props: TypographyProps): JSX.Element => (
+        <Headline
+            style={{ marginTop: 8 }}
+            hash={getHash(props.children?.toString() || 'h5')}
+            TypographyProps={{ variant: 'body2', ...props }}
+        />
+    ),
+    h6: (props: TypographyProps): JSX.Element => (
+        <Headline
+            style={{ marginTop: 8, fontSize: '0.75rem' }}
+            hash={getHash(props.children?.toString() || 'h6')}
+            TypographyProps={{ variant: 'body2', ...props }}
+        />
+    ),
+    a: (props: React.LinkHTMLAttributes<HTMLLinkElement>): JSX.Element => {
+        let tProps;
+        if (props.href?.includes('#')) {
+            tProps = props as TypographyProps<'a'>;
+
+            return (
+                <Typography
+                    component={'a'}
+                    sx={{
+                        fontWeight: 400,
+                        textDecoration: 'none',
+                        color: (theme: Theme) => theme.palette.primary.main,
+                        fontSize: '14px',
+                    }}
+                    {...tProps}
+                />
+            );
+        }
+        if (props.href && (props.href.match(/^http/gi) || props.href.match(/^mailto/gi))) {
+            tProps = props as TypographyProps<'a'>;
+            return <ExternalLink {...tProps} />;
+        }
+        tProps = props as LinkProps;
+        //@ts-ignore
+        return <InternalLink to={props.href} {...tProps} />;
+    },
+    p: (props: TypographyProps): JSX.Element => (
+        <Typography
+            sx={{ ...REGULAR_WIDTH_STYLE, fontSize: '14px', lineHeight: '1.6', m: '15px auto' }}
+            paragraph
+            {...props}
+        />
+    ),
+    ul: (props: BoxProps): JSX.Element => (
+        <Box component={'ul'} sx={{ ...REGULAR_WIDTH_STYLE, fontSize: '14px', m: '15px 0', pl: '60px' }} {...props} />
+    ),
+    li: (props: TypographyProps<'li'>): JSX.Element => (
+        <Typography
+            component={'li'}
+            className={'mdLi'}
+            sx={{ ...REGULAR_WIDTH_STYLE, fontSize: '14px', m: '15px 0' }}
+            {...props}
+        />
+    ),
+    blockquote: (props: TypographyProps<'blockquote'>): JSX.Element => {
+        return (
+            <Typography
+                component={'blockquote'}
+                sx={{
+                    pr: 2,
+                    mb: 1,
+                    borderLeft: ' 4px solid #DDDDDD',
+                    p: '0 15px',
+                    m: '15px 0',
+                    color: '#666666',
+                    ...REGULAR_WIDTH_STYLE,
```

Comment:
These are a lot of hard-coded colors. Can you find the closest color in our theme colors and use those instead?